modules/UI/videolayout/VideoContainer.js

```diff
@@ -402,22 +431,16 @@ export class VideoContainer extends LargeContainer {
             return;
         }
 
-        this._hideVideoBackground();
-
         const [ width, height ]
             = this.getVideoSize(containerWidth, containerHeight);
 
         if ((containerWidth > width) || (containerHeight > height)) {
-            this._showVideoBackground();
-            const css
-                = containerWidth > width
-                    ? { width: '100%',
-                        height: 'auto' }
-                    : { width: 'auto',
-                        height: '100%' };
-
-            this.$videoBackground.css(css);
+            this._backgroundOrientation = containerWidth > width
+                ? ORIENTATION.LANDSCAPE : ORIENTATION.PORTRAIT;
+        } else {
+            this._backgroundOrientation = null;
```

Comment:
i don't follow why the else case here determines that we shouldn't show the background.

`if (containerWidth <= width && containerHeight <= height)` means "don't show the background"? i thought not showing the background was based on the browser?

part of this confusion might come from the fact we overload `_backgroundOrientation` being null to mean "don't show the background"

Reply:
reading the rest of the code got me more confused...i do see later we base the background on the browser type, but this logic seems totally independent? what's the source of truth to show the background or not?

Reply:
I see what you're saying. Before there was one source of truth which was to only show the background when based on some kind of sizing, meaning that based on video sizing there would be some background to display. Now there is essentially two. One is at the higher level in the html by preventing the background container from being created at all. The other way is in this code flow.

Reply:
I've made a slight tweak, essentially so that the background update function exits early instead if the browser is firefox or temasys. Let me know if that helps or something more is needed.

Reply:
yeah i think this feels better